mlfinlab/clustering/feature_clusters.py

```diff
@@ -0,0 +1,142 @@
+"""
+This module creates clustered subsets of features described in the paper Clustered Feature Importance (Presentation
+Slides) by Dr. Marcos Lopez de Prado. https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3517595 and is also explained
+in the book Machine Learning for Asset Managers Snippet 6.5.2 page 84.
+"""
+
+#Imports
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from scipy.spatial.distance import squareform
+from scipy.cluster.hierarchy import linkage, fcluster
+from statsmodels.regression.linear_model import OLS
+
+from mlfinlab.clustering.onc import get_onc_clusters
+from mlfinlab.codependence.codependence_matrix import get_dependence_matrix, get_distance_matrix
+
+
+# pylint: disable=invalid-name
+def get_feature_clusters(X: pd.DataFrame, dependence_metric: str, distance_metric: str = None, linkage_method: str = None,
```

Comment:
Please add typings for return expression (-> list of tuples)

Reply:
done.